src/tracing/trace_event.h

```diff
@@ -0,0 +1,1710 @@
+// Copyright 2015 the V8 project authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SRC_TRACING_TRACE_EVENT_H_
+#define SRC_TRACING_TRACE_EVENT_H_
+
+#include <stddef.h>
+
+#include "v8-platform.h"
```

Comment:
Those macros refer to `v8::internal::tracing::TraceEventHelper`, but that is different from `node::tracing::TraceEventHelper` that I was referring to.